src/main.rs

```diff
@@ -347,14 +401,11 @@ fn main() {
         },
     };
 
-    if let Err(err) = status {
-        if sevctl.quiet {
-            exit(1);
-        }
-        eprintln!("error: {}", err);
-
+    if status.is_err() && sevctl.quiet {
```

Comment:
In this case, wouldn't an error message never be printed?

Reply:
Yes, but that's how it's expected to work with the `--quiet` flag. See where old code just called `exit(1)`

Reply:
Right, but there is no case for `status.is_err() && !sevctl.quiet`, where we could assume that a user would like something printed, no?

Reply:
Sorry, missed this follow up. If you check the full diff, we exit the process when this condition is true. So `status.is_err() && !sevctl.quiet` is implicitly handled by the plain function return value, which prints proper error output and sets proper error code.